src/container.js

```diff
@@ -63,9 +63,9 @@ function injectProperties(container, fn, instance) {
     for (let property in dependencies) {
       instance[property] = container.get(dependencies[property]);
     }
-  }
-  if (instance.postConstruct !== undefined) {
-    instance.postConstruct();
+    if (instance.postConstruct !== undefined) {
```

Comment:
This is precisely what I was thinking of with #88: the ability to inject services and have a hook called once they're all initialized.